### PR TITLE
Fix .env

### PR DESCRIPTION
**Pull request by ghost**
this should fix the issue with .env

### CODE REVIEW

**Comment by ghost** [comment]
https://github.com/cognitedata/generator-create-react-redux-app/issues/17

File: generators/app/templates/package.json [hunk 1]
```diff
@@ -27,10 +27,10 @@
     "redux-mock-store": "^1.2.3"
   },
   "scripts": {
-    "start": "cognite-react-scripts start",
-    "build": "cognite-react-scripts build",
-    "test": "cognite-react-scripts test --env=jsdom",
-    "eject": "cognite-react-scripts eject",
+    "start": "NODE_PATH=src/ cognite-react-scripts start",
```
Comment (ghost) [inline generators/app/templates/package.json:30]: also i dont think the current package.json has `cognite-react-scripts`

Comment (halvard-cognite) [reply generators/app/templates/package.json:30]: Yes it does. I have been working on it all morning :)

Comment (halvard-cognite) [reply generators/app/templates/package.json:30]: @cognite/react-scripts

Comment (halvard-cognite) [reply generators/app/templates/package.json:30]: `@cognite/react-scripts`

Comment (ghost) [reply generators/app/templates/package.json:30]: ahh i see it sorry!

**Comment by ghost** [comment]
@halvard-cognite should i merge?

**Comment by ghost** [comment]
@halvard-cognite i will update the react-scripts to load the src file directly without using .env then :)

**Pull request closed without merging**